app/editor/src/features/content/form/ClipDirectoryTable.tsx

```diff
@@ -28,15 +28,21 @@ export const ClipDirectoryTable: React.FC<IClipDirectoryTableProps> = ({
 }) => {
   const { values } = useFormikContext<IItemModel>();
 
+  const [activeId, setActiveId] = React.useState<number>();
+
   return (
     <styled.ClipDirectoryTable>
       <GridTable
+        activeId={activeId}
         paging={{ showPaging: false }}
         columns={clipDirectoryColumns(onDelete, onSelect, onDownload, onAttach, values)}
         header={ClipDirectoryFilter}
         data={data}
         className="file-table"
-        onRowClick={(row) => navigate(row.original)}
+        onRowClick={(row) => {
+          setActiveId(row.index);
```

Comment:
This doesn't work yet.  I need to modify the GridTable to allow for either number or string to identify a row.  Another day another PR.